DEV: attempts to fix various leaks

jjaffeux opened the pull request.

eviltrout commented on test/javascripts/test_helper.js:
```diff
@@ -105,6 +105,8 @@ function resetSite(siteSettings, extras) {
   Site.resetCurrent(Site.create(siteAttrs));
 }
 
+let baseTransalations = I18n.translations[I18n.locale].js || {};
```
This isn't copying them right? It's a reference to that object. So if that object is mutated setting it back does nothing?

jjaffeux replied:
Oh yes I changes the impelmentation of this at the end and didnt check but yes you are right I think. This one doesn’t matter that much anyways, it mosty prevents to carry locales over time, will come up with something better.

jjaffeux commented:
@eviltrout I have removed the I18NN change because it's not needed, and also I changed every scheduleOnce with anon function to schedule because this is broken. A scheduleOnce with an anonymous function will work as a simple schedule as there's no way it can know it's the same thing.

danielwaterworth commented:
> A scheduleOnce with an anonymous function will work as a simple schedule as there's no way it can know it's the same thing.

Aside from doing toString on it :trollface:
